README.md

```diff
@@ -230,6 +233,80 @@ label.bonMotStyleName = @"MyHeadline";
 
 - Use the inspectable properties of common UIKit elements as in the [Interface Builder section](#storyboard-and-xib-integration).
 
+# BonMot 3 → 4 Migration Guide
+
+BonMot 4 is a major update, but there are some common patterns that you can use to ease the transition. Note that this is mostly for Swift projects that were using BonMot 3. BonMot 4 has only limited [support for Objective-C](#objective-c-compatibility), so please check that section before attempting to upgrade if you need to maintain Objective-C compatibility.
+
```

Comment:
I'd call out that the primary design change between 3 and 4 is the separation of style and an applied style. You mention it below a bit, but calling it out up front may get the point across better.